profiles/ighoshsubho.json

```
@@ -0,0 +1,6 @@
+{
+    "image": "https://avatars.githubusercontent.com/u/93722719?v=4",
+    "issueId": 1862,
+    "name": "Subho Ghosh",
+    "username": "ighoshsubho"
+  }
```

Missing an endline. Without this endline, there could be issues with some IDEs and configurations.
```suggestion
}

```